core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -79,6 +79,15 @@
     private boolean checkConsistency = false;
     private boolean saveKeyValues;
     private TableId tableId;
+    private AccumuloClient _client;
+
+    Builder(AccumuloClient client) {
+      this._client = client;
+    }
+
+    Builder() {
+
+    }
```

Comment:
Should only have one constructor and one build method to prevent coding errors.  Since you have access to the AccumuloClient object at the time you are calling the method, I think it would be best to go with the `Builder(AccumuloClient client)` constructor and then end it with the `build()` method.  This will only pass in the client once and prevent it from being null when calling `build()`.

Reply:
One issue with this is that there are ~15 classes that already use the pre-existing no-parameter constructor. Perhaps I should find a different way to pass the AmpleImpl Accumulo client into the TabletsMetadata builder object.

Reply:
I might be able to refactor those classes to use the new Builder(Accumulo client) constructor.

Reply:
That is OK.  I should only be 1 or 2 lines per class.

Reply:
In the latest push: Refactored the other classes to use the new constructor. Cleaned up/streamlined unit test in MetadataIT.